src/sass/partials/_button.scss

```diff
@@ -0,0 +1,34 @@
+.button {
+  width: 130px;
+  height: 44px;
+
+  background-color: transparent;
+  color: white; //placeholders
+  border: 1px solid white;
+  border-radius: 5px;
+
+  font-size: 12px;
+  font-weight: 500;
+  line-height: 16px;
+
+  padding: 6px 13px;
+
+  @media screen and (min-width: 768px) {
+    width: 152px;
+  }
+
+  &:hover,
+  &:focus {
+    background-color: #ff6b01;
+    border-color: #ff6b01;
+  }
+
+  &__modal {
+    width: 110px;
+    color: black;
+
+    @media screen and (min-width: 768px) {
+      width: 125px;
+    }
+  }
+}
```

Comment:
background:
border:
border-radius:
color: 
font-weight:
line-height:
&hover &focus
no need to add more things now we gonna use elements styles later
instead of width and height implement padding's